### DISCUSSION TITLE
v0.9.7

### DISCUSSION THREAD

**Discussion by majestrate** [oxen-io/lokinet#1777] [General]
# lokinet v0.9.7

bugfix release for clients.

client side fixes:

* work around volatile network path build success rate, prevent clients from becoming stuck in an edge case state from which it used to not recover from.
* bundle bootstrap.signed by default instead of requiring it be fetched externally
* logging subsystem refactor to use source_location
* fix issue with older systemd-resolved not being able to set dns
* lokinet will now load config files in lexigraphical order
* remove json logger

ci pipeline updates:

* basically totally redo the ci pipeline docker stuff

<hr /><em>This discussion was created from the release <a href='https://github.com/oxen-io/lokinet/releases/tag/v0.9.7'>v0.9.7</a>.</em>

**Comment by Destroyinator69420** [comment]
When will you produce a windows binary? I am stuck on windows for school.

**Reply by majestrate** [reply]
yea
